docs/tutorials/07_leveraging_qiskit_runtime.ipynb

```diff
@@ -24,6 +24,15 @@
     "Let's see how you can leverage the runtime on a simple chemistry example: Finding the ground state energy of the lithium hydrate (LiH) molecule at a given bond distance."
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### *Important: Version requirements*\n",
+    "\n",
+    "*This notebook requires Qiskit Terra 0.18.x and Qiskit Nature 0.3.x.*"
```

Comment:
Can we not say a minimum of 0.18.0 and 0.3.0. I imagine by the time Nature 0.3.0 is out that Terra will be on the next version

Reply:
done in 99a2433